packages/react-server/core/TritonAgent/Request.js

```diff
@@ -101,6 +102,27 @@ Request.prototype.type = function (type) {
 	return this;
 }
 
+Request.prototype.noCacheResponse = function () {
+	this._noCacheResponse = true;
+	return this;
+}
+
+Request.prototype.getCacheEntry = function() {
+	if(this._noCacheResponse) {
+		return new CacheEntry(
+			null,
+			this._getCacheAffectingData(),
+			this._cacheWhitelist
+		)
+	} else {
+		return this._cache.entry(
+			this._getCacheAffectingData(),
+			SERVER_SIDE, // createIfMissing
+			this._cacheWhitelist
+		)
+	}
+}
+
```

Comment:
Boy, this PR would be a lot easier to grok if it had the _use_ of these new methods along side of it. :-/

I made [this comment](https://github.com/redfin/react-server/pull/52/files#r55273860) while you were making this one.  Does that clear things up?